bin/check_tron_jobs.py

```diff
@@ -111,6 +113,16 @@ def get_relevant_run(job_runs):
     return None
 
 
+def is_job_stuck(job_runs):
+    for run in job_runs['runs']:
+        run_state = run.get('state', 'unknown')
+        if run_state in ["running"]:
+            return False
+        if run_state in ["queued", "cancelled"]:
+            return True
```

Comment:
This isn't really a robust way to detect if a job is stuck.

What if a user simply runs 'tronctl stop' on a job and it becomes canceled? I would not want to email them telling them that it is stuck just because of that.

Reply:
Also `queued` is not enough to detect if a job is stuck or not if a user has allowed `allow_overlapping`

Reply:
Good point, I didn't take stopping job into account. 
But if a user has allowed allow_overlapping, user would expect one job run might longer than the scheduled interval, right? In this case, do we still concern the job being stuck?

Reply:
Yes, we still need to be concerned about a job being stuck.

"stuck" really is the wrong word. What users really want is: "Has it been too long since my job has completed successfully? (I don't care what the "state" is in the tron api)"